model-optimizer/mo/ops/pooling.py

```diff
@@ -105,14 +117,26 @@ def pool_infer(node: Node):
             node['window'] = np.zeros(len(input_shape), dtype=np.int64)
             node.window[node.spatial_dims] = input_spatial_shape
 
+        if not node.has_valid('dilation'):
+            node['dilation'] = np.ones(len(input_shape))
+
+        if not node.has_valid('axis'):
+            node['axis'] = 0
+
+        if not node.has_valid('index_element_type'):
+            node['index_element_type'] = np.int64
```

Comment:
this attribute must fw specific and must be set in extractor. Please see `TopK`
Are we sure that for TF2 int64 is suitable? For ONNX, the same question.

Reply:
ONNX indices always have int64 type. I can move the setting of this attribute to the ONNX extractor, but I still need to provide a default value for other frameworks. The default value is int64 according to our MaxPool-8 spec.

TF MaxPool has no second output, so this attribute will have a default value.

Reply:
https://www.tensorflow.org/api_docs/python/tf/nn/max_pool_with_argmax has two outputs
do we have extractor for this?

Reply:
Ok now I understand what you mean